packages/wrangler/src/__tests__/dev.test.tsx

```diff
@@ -439,16 +439,36 @@ describe("wrangler dev", () => {
 				main: "index.js",
 			});
 			fs.writeFileSync("index.js", `export default {};`);
-			mockGetZones("111.222.333.some-host.com", []);
-			mockGetZones("222.333.some-host.com", []);
-			mockGetZones("333.some-host.com", [{ id: "some-zone-id" }]);
-			await runWrangler("dev --host 111.222.333.some-host.com");
-			expect((Dev as jest.Mock).mock.calls[0][0]).toEqual(
-				expect.objectContaining({
-					host: "111.222.333.some-host.com",
-					zone: "some-zone-id",
+
+			msw.use(
+				rest.get("*/zones", (req, res, ctx) => {
+					return res(
+						ctx.status(200),
+						ctx.json({
+							success: true,
+							errors: [],
+							messages: [],
+							result:
+								req.url.searchParams.get("name") === "111.222.333.some-host.com"
```

Comment:
Nit: I'd declare this as a variable below `rest.get("*/zones", (req, res, ctx) => {` and use if/else if/else instead of a double ternary

Reply:
The if/else breaks, I could possibly make a reassignment of a variable through if/else

Reply:
Done.